index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Ali Kimbrell | Software Developer</title>
+		<meta name="description" content="">
+		<meta name="author" content="Ali Kimbrell">
+		<meta charset="utf-8">
+		<link rel="stylesheet" href="/bower_components/bootstrap/dist/css/bootstrap.min.css">
+		<link rel="stylesheet" href="css/quiz.css">
+	</head>
+
+	<body>
+		<div id="wrapper">
+		<div id="background">
+			<header>
+				<!-- NAVBAR -->
+				<nav class="navbar navbar-default">
+				  <div class="myMenu container-fluid">
+				    <!-- Brand and toggle get grouped for better mobile display -->
+				    <div class="navbar-header">
+				      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+				        <span class="sr-only">Toggle navigation</span>
+				        <span class="icon-bar"></span>
+				        <span class="icon-bar"></span>
+				        <span class="icon-bar"></span>
+				      </button>
+				      <!-- <a class="navbar-brand" href="#">Brand</a> -->
+				    </div>
+				    <!-- Collect the nav links, forms, and other content for toggling -->
+				    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+				      <ul class="nav navbar-nav navbar-right">
+				        <li><a href="#">Technologies</a></li>
+				        <li><a href="#">Portfolio</a></li>
+				        <li><a href="#">About</a></li>
+				        <li><a href="#">Contact</a></li>
```

Comment:
So, you have this nice navbar going but the links are to things on the index page. I know this is just a first draft of you portfolio, but it would be nice to see these links do something or not be there until they do. Your master branch should contain working, deploy-ready code.